features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Resources/components/AddOperation.jsx

```diff
@@ -319,4 +328,4 @@ AddOperation.propTypes = {
     operationsDispatcher: PropTypes.func.isRequired,
 };
 
-export default React.memo(AddOperation);
+export default React.memo(injectIntl(AddOperation));
```

Comment:
Shall we check whether `react-intl` library provides a hook to inject these into props, AFAIR back then they discussed to provide a react hook instead of this HOC

Reply:
I think they have included it since react v16.3. 
https://github.com/formatjs/react-intl/blob/master/docs/Upgrade-Guide.md#new-useintl-hook-as-an-alternative-of-injectintl-hoc

Will check.

Reply:
https://github.com/wso2/carbon-apimgt/pull/8272/commits/d62db219d1ca80d6d81986e375db039236ef5da1